cmd/chisel/cmd_info_test.go

```diff
@@ -0,0 +1,250 @@
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    baz:
+        essential:
+            - libpkg_libs
+            - mypkg_bar
+            - mypkg_foo
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+}, {
+	summary: "Different packages, multiple slices of same packages",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo", "libpkg", "mypkg_baz"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    baz:
+        essential:
+            - libpkg_libs
+            - mypkg_bar
+            - mypkg_foo
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+---
+package: libpkg
+archive: ubuntu
+slices:
+    libs:
+        contents:
+            /usr/lib/libpkg.so*: {}
+`,
+}, {
+	summary: "Same package, multiple slices",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo", "mypkg", "mypkg_baz"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    bar:
+        essential:
+            - mypkg_foo
+        contents:
+            /bin/bar: {}
+            /etc/bar.conf: {text: TODO, mutable: true, arch: riscv64}
+            /lib/*-linux-*/bar.so: {arch: [amd64, arm64, i386]}
+            /usr/bin/bar: {symlink: /bin/bar}
+            /usr/bin/baz: {copy: /bin/bar}
+            /usr/lib/bar*.so: {}
+            /usr/share/bar/*.conf: {until: mutate}
+        mutate: |
+            dir = "/usr/share/bar/"
+            conf = [content.read(dir + path) for path in content.list(dir)]
+            content.write("/etc/bar.conf", "".join(conf))
+    baz:
+        essential:
+            - libpkg_libs
+            - mypkg_bar
+            - mypkg_foo
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+}, {
+	summary: "Same slice, appearing multiple times",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo", "mypkg_foo", "mypkg_foo"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+}, {
+	summary: "No slices found",
+	input:   infoRelease,
+	query:   []string{"foo", "bar_foo"},
+	err:     `no slice definitions found for: "foo", "bar_foo"`,
+}, {
+	summary: "Some slices found, others not found",
+	input:   infoRelease,
+	query:   []string{"foo", "mypkg_foo", "bar_foo"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
+	err: `no slice definitions found for: "foo", "bar_foo"`,
+}, {
+	summary: "No args",
+	input:   infoRelease,
+	err:     "the required argument `<pkg|slice> (at least 1 argument)` was not provided",
+}, {
+	summary: "Empty, whitespace args",
+	input:   infoRelease,
+	query:   []string{"", "    "},
+	err:     `no slice definitions found for: "", "    "`,
+}, {
+	summary: "Bad format slices",
```

Comment:
```suggestion
	summary: "Ignore invalid slice names",
```

Reply:
I am now wondering whether it would be better to return an error if there is an invalid slice name because that is probably unintended. What do you think? Let's discuss but  let's not change the implementation yet.

Reply:
Since `info` takes a bunch of search terms, it would probably be nice to report all the matches found. We are already returning the unmatched ones as an error. The invalid names would be reported there -- albeit, not as "invalid".

Reply:
Discussed offline: we would need to return two types of errors for this which is not very conventional. Let's keep it as it is for now.